app/assets/javascripts/ckeditor/plugins/pasteFromGoogleDoc/plugin.js

```diff
@@ -0,0 +1,405 @@
+			/**

```

Comment:
Third party scripts like this would conventionally be under vendor/assets/javascripts. In any case, we'll need to add it to the `config.assets.precompile` list. The gem suggests adding all of ckeditor/\*, and we used to do that, but I had to remove it to fix #36. You could add all of vendor/assets/javascripts/* if you move it there, or otherwise add this file only.